element/index.js

```diff
@@ -29,3 +29,15 @@ export { render };
  * A base class to create WordPress Components (Refs, state and lifecycle hooks)
  */
 export { Component };
+
+/**
+ * Renders a given element into a string
+ *
+ * @param {wp.Element} element Element to render
+ * @return {String}            HTML
+ */
+export function renderToString( element ) {
```

Comment:
We should use `react-dom/server`'s `renderToStaticMarkup` which, unlike `renderToString`, includes no checksum or `reactid`.